src/main/java/it/pagopa/transactions/utils/TransactionEventCode.java

```diff
@@ -2,7 +2,8 @@
 
 public enum TransactionEventCode {
 
-    TRANSACTION_INITIALIZED_EVENT("TRANSACTION_INITIALIZED_EVENT");
+    TRANSACTION_INITIALIZED_EVENT("TRANSACTION_INITIALIZED_EVENT"),
+    TRANSACTION_AUTHORIZED_EVENT("TRANSACTION_AUTHORIZED_EVENT");
```

Comment:
At this step the transaction is not authorized, but the request has been executed, so better `TRANSACTION_AUTHORIZATION_REQUESTED_EVENT`? or something like that?